File: behavioral/python/state/TRYAccount.py
from AccountA import AccountA

class TRYAccount(AccountA):
    def withdrawMoney(self, moneyAmount: int) -> None:
        print("Withdraw money of TRY account. " + str(moneyAmount) + "TRY")
    def depositMoney(self, moneyAmount: int) -> None:
        print("Deposit money of TRY account. " + str(moneyAmount) + "TRY")
    def getAccountName(self) ->str:
        return "TRY"

File: behavioral/jv/state/src/TRYAccount.java
public class TRYAccount implements AccountA{
    @Override
    public void withdrawMoney(int moneyAmount) {
        System.out.println("Withdraw money of TRY account. " + Integer.toString(moneyAmount) + "TRY");
    }

    @Override
    public void depositMoney(int moneyAmount) {
        System.out.println("Deposit money of TRY account. " + Integer.toString(moneyAmount) + "TRY");
    }

    @Override
    public String getAccountName() {
        return "TRY";
    }
}
